alvr/server_core/src/tracking/face.rs

```diff
@@ -142,6 +144,10 @@ impl FaceTrackingSink {
                     self.append_packet_vrcft(b"LipHtc\0\0", &arr);
                 }
 
+                if let Some(arr) = face_data.pico_face_expression {
```

Comment:
Here, instead of the pico_face_expression, you should match fb or pico prefix based on the length of fb_face_expression